app/controllers/citizens/other_assets_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Citizens
+  class OtherAssetsController < BaseController
+    def show
+      declaration
+      @form = Citizens::OtherAssetsForm.new(current_params)
```

Comment:
Actually, the latest version of `BaseForm` has fixed the issue with using `...Form.new(model: model)` so my original comment stands. You can replace this line with:
```ruby
@form = Citizens::OtherAssetsForm.new(model: legal_aid_application)
```
And remove the `current_params` params